src/js/constants/ServiceSchema.js

```diff
@@ -45,7 +45,7 @@ let SERVICE_SCHEMA = {
           type: 'number',
           default: 1,
           getter: function (service) {
-            return (service.getInstancesCount() || 0);
+            return (service.getInstancesCount() || 0) + '';
```

Comment:
Not sure why this has to be an string, but fair enough, there will be a reason for that.